### ISSUE TITLE
Prefer single quotes for import directives

### ISSUE THREAD

**Issue by Mathorga** [dart-lang/lints#173]
**Describe the rule you'd like to see added and to what rule set**
Include as much detail as you can. 

Prefer single quotes for import directives only. This rule should not be incompatible with `prefer_double_quotes`, but override it for import directives.
This is useful for compliance with community standards for import directives, while not conflicting with rules that allow or even enforce the use of double quotes for the rest of the code.
If accepted, the rule should be added to the `recommended` rule set, as it should not affect package scoring and is not as essential to app and package development.

BAD:
import "package:flutter/material.dart";

void main {
  const String testString = "test";
  runApp(const MyApp());
}

GOOD:
import 'package:flutter/material.dart';

void main {
  const String testString = "test";
  runApp(const MyApp());
}

**Additional context**
Add any other considerations or context here.

**Comment by devoncarew** [comment]
Thanks for the proposal! This repo tracks adding existing lints to the `core` and `recommended` set of lints. I gather your suggestion is an idea for a new lint? If so, I'd re-file this at dart-lang/linter.

**Comment by Mathorga** [comment]
Yes, I'm sorry, my bad, this is a suggestion for a new lint, it should be re-filed at dart-lang/linter. Thank you very much!
